### PR TITLE
fix(rss3): new UI design for donation page, and support cheers.io

### PR DESCRIPTION
**Pull request by UncleBill**
closes [#MASK-761](https://mask.atlassian.net/browse/MASK-761)

image

### CODE REVIEW

**Review by guanbinrui — DISMISSED** [review]

**Comment by guanbinrui** [comment]
@DimensionDev/qa

**Comment by wenluomask** [comment]
Checked.

File: packages/plugins/RSS3/src/SNSAdaptor/TabCard.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    const { value: footprints = EMPTY_LIST, loading: loadingFootprints } = useFootprints(userAddress)
+
+    if (!addressName) return null
+
+    const isDonation = type === TabCardType.Donation
+
+    const summary =
+        isDonation && !loadingDonations ? (
+            <Typography color="textPrimary" component="span">
+                Total {donations.length} Grants
```
Comment (guanbinrui) [inline packages/plugins/RSS3/src/SNSAdaptor/TabCard.tsx]: `useI18N()`

File: packages/plugins/RSS3/src/SNSAdaptor/components/StatusBox.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            <Box className={classes.statusBox}>
+                <CircularProgress />
+            </Box>
+        )
+    }
+
+    if (empty) {
+        return (
+            <Box className={classes.statusBox}>
+                <Typography color="textPrimary">No data.</Typography>
```
Comment (guanbinrui) [inline packages/plugins/RSS3/src/SNSAdaptor/components/StatusBox.tsx]: `useI18N()`

Comment (guanbinrui) [inline packages/plugins/RSS3/src/SNSAdaptor/components/StatusBox.tsx]: `useI18N()`

Comment (UncleBill) [reply packages/plugins/RSS3/src/SNSAdaptor/components/StatusBox.tsx]: All missed i18 added.

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**